setup.cfg

```diff
@@ -38,13 +38,17 @@ install_requires =
     # (first version with universal2 wheels available)
     numpy==1.21.0; python_version=='3.8' and platform_machine=='arm64' and platform_system=='Darwin'
     numpy==1.21.0; python_version=='3.9' and platform_machine=='arm64' and platform_system=='Darwin'
+    
+    # If a C extension uses npymath, then Windows needs a bugfix in numpy>=1.14.1
```

Comment:
Why do the comments and the pins not match?  Comments says >= 1.14.1 but pin is for 1.14.0. Should pin be 1.14.5, which is the last release of 1.14 which has the same C interface as 1.14.0?

Reply:
as for 1.14.5 as the preferred choice, i’ll defer to others on that

Reply:
I had a look at this for another issue recently, and it should indeed be fine to compile against the latest bugfix release of a minor release (so `1.14.5`) and then use it with all `1.14.x` releases.

Reply:
fixed comment and bumped to 1.14.5